app/helpers/comments_helper.rb

```diff
@@ -84,4 +84,20 @@ def comment_for(model_name)
   rescue ActiveRecord::RecordInvalid
     respond_not_saved
   end
+
+  # rubocop:disable MethodLength
+  def user_created_data?(id, data_type)
```

Comment:
So it looks like `user_created_data?` is only used in `comment_deletable?`. I would put a private above this method.

Reply:
sure 👍 . Do you think there might be other methods inside this helper that need to be private ?

Reply:
I would double check to make sure, I don't know off the top of my head!

Reply:
I looked into the file and moved the methods that were privat under that scope. Let me know what you think